CMakeLists.txt

```diff
@@ -345,6 +345,7 @@ if(NOT NO_OPENGL AND NOT ANDROID AND NOT IOS)
 endif()
 if(NOT NO_PTEX)
    find_package(PTex 2.0)
+   find_package(Zlib 1.2)
```

Comment:
CMake ships with a FindZLIB module, but you have to set the root location... (-DZLIB_ROOT="...")